worker_container_definition.json

```diff
@@ -0,0 +1,76 @@
+[
+  {
+    "name": "${task_name}",
+    "image": "${image}:latest",
+    "memory": 512,
+    "networkMode": "bridge",
+    "command": ${worker_command},
+    "logConfiguration": {
+      "logDriver": "json-file"
+    },
+    "portMappings": [
+      {
+        "containerPort": ${task_port},
+        "hostPort": 0
+      }
+    ],
+    "environment": [
+      {
+        "name": "RAILS_ENV",
+        "value": "${rails_env}"
+      },
+      {
+        "name": "ROLLBAR_ENV",
+        "value": "${environment}"
+      },
+      {
+        "name": "ROLLBAR_ACCESS_TOKEN",
+        "value": "${rollbar_access_token}"
+      },
+      {
+        "name": "SECRET_KEY_BASE",
+        "value": "${secret_key_base}"
+      },
+      {
+        "name": "RAILS_LOG_TO_STDOUT",
+        "value": "true"
+      },
+      {
+        "name": "RAILS_SERVE_STATIC_FILES",
+        "value": "true"
+      },
+      {
+        "name": "DATABASE_URL",
+        "value": "postgres://${database_user}:${database_password}@${database_url}:5432/${project_name}_${environment}?template=template0&pool=5&encoding=unicode"
+      },
+      {
+        "name": "ELASTICSEARCH_URL",
+        "value": "https://${elastic_search_url}:443"
+      },
+      {
+        "name": "ELASTICSEARCH_AWS_SIGNING",
+        "value": "true"
+      },
+      {
+        "name": "AWS_ELASTICSEARCH_REGION",
+        "value": "${aws_elasticsearch_region}"
+      },
+      {
+        "name": "AWS_ELASTICSEARCH_KEY",
+        "value": "${aws_elasticsearch_key}"
+      },
+      {
+        "name": "REDIS_URL",
+        "value": "${redis_url}"
+      },
+      {
+        "name": "PP_TRANSACTIONS_BY_CHANNEL_TOKEN",
+        "value": "${pp_transactions_by_channel_token}"
+      },
+      {
+        "name": "PP_USER_SATISFACTION_TOKEN",
+        "value": "${pp_user_satisfaction_token}"
+      }
```

Comment:
I think we probably want to add `GOOGLE_API_JSON_KEY` and `google_api_json_key` here too.

Reply:
👍